packages/fogbender-element/src/FogbenderProvider.tsx

```diff
@@ -1,25 +1,32 @@
+import { createContext, provide } from "component-register";
 import { createNewFogbender, Fogbender } from "fogbender";
-import type { JSX, Accessor, Component } from "solid-js";
-import { createContext, useContext, createSignal, createEffect } from "solid-js";
+import { customElement } from "solid-element";
+import type { JSX } from "solid-js";
 
 export interface FogbenderProviderProps {
-  fogbender?: Accessor<Fogbender>;
-  children?: JSX.Element;
+  fogbender?: Fogbender;
+  children?: JSX.Element[];
 }
 
-export const FogbenderContext = createContext<Accessor<Fogbender>>();
+interface Context {
```

Comment:
why do we need this type?

Reply:
Because we are exporting context and if we were doing this without defining the interface in this file we were getting this error 
![image](https://user-images.githubusercontent.com/47587853/192138215-730d26a2-6b80-4726-9d51-86ba40940705.png)
When in tsconfig` declaration : true`
And this was true in other packages of fogbender tsconfig files